types/ioredis-mock/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
+            true,
+            {
+                "mode": "code",
+                "errors": [["NeedsExportEquals", false]]
+            }
+        ]
+    }
+}
```

Comment:
The rules needs to be removed (see [here](https://github.com/DefinitelyTyped/DefinitelyTyped#user-content-linter-tslintjson))

```suggestion
{
    "extends": "dtslint/dt.json"
}
```

Reply:
@nicolas377 I tried making it work without this rule, but didn't manage to do it. Is it even possible? I noticed many other typings which also seem to need this rule. The command line told me to add this rule if I think this error is wrong:

![image](https://user-images.githubusercontent.com/326935/130593927-aa505ff0-8fa2-4423-b90b-50da156fbe51.png)